lib/api.rb

```diff
@@ -0,0 +1,20 @@
+module Api
```

Comment:
Yeah, I like more unique namespaces.  Everything within that namespace shouldn't care about the ManageIQ namespace as it can use `API::Server` from within the ManageIQ namespace.  I don't know about `API::Server`... I'll leave that to you... I hear you're good at naming things 😉